src/docker-osx-dev

```diff
@@ -409,7 +409,12 @@ function configure_docker_machine {
   DOCKER_MACHINE_DRIVER_NAME=$(inspect_docker_machine "{{.DriverName}}")
 
   DOCKER_HOST_SSH_URL="$DOCKER_HOST_USER@$DOCKER_HOST_IP"
-  DOCKER_HOST_SSH_KEY="$DOCKER_MACHINE_STORE_PATH/id_rsa"
+  DOCKER_HOST_SSH_KEY="$DOCKER_MACHINE_STORE_PATH/machines/$DOCKER_MACHINE_NAME/id_rsa"
+
+  if [[ ! -f $DOCKER_HOST_SSH_KEY ]]; then
```

Comment:
Ah fair enough, so the value of StorePath has now changed entirely on 0.5.1.

The location of the key still remains the same. Since 0.5.1 suggest [to not use](https://github.com/docker/machine/blob/34f818527debef02f5a3102f198e44fb911433ae/libmachine/auth/auth.go#L15) `StorePath`, I think we just keep the backup conditional and leave a note in the code.